package.json

```diff
@@ -44,19 +44,19 @@
   "devDependencies": {
     "@commitlint/cli": "^17.0.3",
     "@commitlint/config-conventional": "^17.0.3",
-    "@firebolt-js/openrpc": "rdkcentral/firebolt-openrpc#next",
```

Comment:
We shouldn't commit `file:` references to package.json, since it depends on everyone having the same file structure.

Change this to `rdkcentral/firebolt-openrpc#feature/c-language` and it will point to the latest commit from GitHub, and it will work for everyone.

Reply:
Don't forget to do an `npm  install` before you commit, so the package-lock.json is included (required for CI to work)